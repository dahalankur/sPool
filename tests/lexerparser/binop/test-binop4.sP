# parentheses for arith
(1 + 2) * 90 - (1234 / (0 / 6))
