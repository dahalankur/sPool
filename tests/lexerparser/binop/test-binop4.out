PROGRAM {[ 
    BINOP(BINOP(BINOP(INT(1), PLUS, INT(2)), TIMES, INT(90)), MINUS, BINOP(INT(1234), DIV, BINOP(INT(0), DIV, INT(6))))]}
